packages/calcite-components/src/components/button/button.tsx

```diff
@@ -279,7 +279,13 @@ export class Button
           }}
           disabled={childElType === "button" ? this.disabled || this.loading : null}
           download={
-            childElType === "a" && (this.download === "" || this.download) ? this.download : null
+            childElType === "a"
```

Comment:
with this change, when `download` is true, we set the prop as `""` instead.
not required for Stencil, but this is needed for Lit as `download` attribute is typed as `string` rather than `string | boolean` in Lumina for technical reasons (it's one few such special attributes for legacy dom reasons)